tests/Resources/test_passwords.py

```diff
@@ -0,0 +1,167 @@
+import base64
+import json
+import unittest
+
+import requests_mock
+
+from frontstage_api import app
+
+
+url_get_token = app.config['OAUTH_TOKEN_URL']
+url_reset_password_request = app.config['RAS_PARTY_RESET_PASSWORD_REQUEST']
+url_verify_token = app.config['RAS_PARTY_VERIFY_PASSWORD_TOKEN']
+url_change_password = app.config['RAS_PARTY_CHANGE_PASSWORD']
+
+
+class TestPasswords(unittest.TestCase):
```

Comment:
Would this test be more readable if it was split out into `TestChangePassword`, `TestRequestPasswordChange`, `TestVerifyPasswordToken`?

Reply:
^ This would be good for the other sets of tests as well, the secure messaging test class has ~25 tests in it for example